### ISSUE TITLE
Training parameters on FCN-8s cityscape

### ISSUE THREAD

**Issue by xuwh15** [phillipi/pix2pix#147]
Hello @phillipi, I am trying to reproduce the results using FCN-8s using pytorch, what hyper parameters did you use for training FCN-8s on the cityscape dataset? 
Looking forward to your reply.
Thank you.

**Comment by junyanz** [comment]
@tinghuiz

**Comment by xuwh15** [comment]
@tinghuiz Thank you for replying. 
Also, I want to ask what is your base learning rate and how many epochs have you trained the network?
Looking forward to your reply.

**Comment by tinghuiz** [comment]
You can find the Caffe solver file I used here: http://people.eecs.berkeley.edu/~tinghuiz/projects/pix2pix/fcn-8s-cityscapes/solver.prototxt. It should contain all the details about the training setup. I don't recall the number of epochs I trained it for though.

**Comment by xuwh15** [comment]
OK, thank you @tinghuiz
